docs/conf/portland/2018/press-kit.rst

```diff
@@ -0,0 +1,49 @@
+:template: 2018/about.html
+
+Press Kit
+=========
+
+We love that you're supporting Write the Docs by writing about the conference, sharing news about it on social media, or asking your manager if you can attend. We're trying to make it as easy as possible for you to do any of the above, so here are some sample tweets, emails and images you can use.
+
+Sample Tweets
+--------------
+
+TODO
+
+Sample Emails
+--------------
+
+Convince Your Employer
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Do you need help justifying why your employer should send you to the Write the Docs? Take a look at the letter below and tweak it as required.
+
+  FROM: [your name]
+  TO: [your employer or manager's name]
+  RE: Continuous Improvement and Education
+
+  I'd like to attend Write the Docs in Portland, 7-8 May 2018. A three-day event that explores the art and science of documentation. The conference covers topics related to documentation in the software industry.
+
+  Write the Docs brings everyone who writes the docs together in the same room: Writers, Developers, Developer Relations, and Support Staff, which makes it an ideal networking opportunity.
+
+  There is a single talk track, a parallel unconference event and a writing day. The `sessions from 2017 <http://www.writethedocs.org/conf/na/2017/speakers/>` provide a reasonable idea of what to expect.
+
+  Cost:
+
+  * ticket $450
+  * travel
+  * accomodation
+
+  Benefits:
+
+  TODO
+
+  TODO
+
+  Sincerely,
+  [your name]
+
+Images and Logos
+-------------------
```

Comment:
Should at least point to the resources repo.

Reply:
Well yeah, still have a number of todos to do :-D